src/data/fintype/basic.lean

```diff
@@ -98,8 +143,8 @@ by simp [eq_univ_iff_forall, finset.nonempty]
   s ∩ univ = s :=
 by rw [inter_comm, univ_inter]
 
-@[simp] lemma piecewise_univ [∀i : α, decidable (i ∈ (univ : finset α))]
-  {δ : α → Sort*} (f g : Πi, δ i) : univ.piecewise f g = f :=
+@[simp] lemma piecewise_univ [∀ i : α, decidable (i ∈ (univ : finset α))]
```

Comment:
```suggestion
@[simp] lemma piecewise_univ [Π i : α, decidable (i ∈ (univ : finset α))]
```
if you're touching this anyway - with the rule of thumb being "the pretty-printer / doc-gen will use `Π` not `∀` for anything that isn't `Prop`, so the source code should too".